tests/automation/setup/open.ts

```diff
@@ -33,20 +36,24 @@ export async function openApp(windowsToCreate: number) {
 
   const array = [...multisToUse];
   const toRet = [];
+  // not too sure why, but launching those windows with Promise.all triggers a sqlite error...
```

Comment:
love it 😂 Maybe there is some singleton logic going on?

Reply:
possibly yeah